activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -373,6 +388,23 @@ def group!(*args) # :nodoc:
     #
     #   User.order('name DESC, email')
     #   # SELECT "users".* FROM "users" ORDER BY name DESC, email
+    #
+    # This could be a source of SQL injection, so only strings composed of plain
+    # column names and simple function(column_name) expressions with optional
+    # ASC/DESC modifiers are allowed.
```

Comment:
```suggestion
    # column names and simple <code>function(column_name)</code> expressions 
    # with optional +ASC+/+DESC+ modifiers are allowed.
```

Also, what do you think about moving this paragraph up above the code examples?

Reply:
I thought this location transitioned nicely to Arel, and meant they had an understanding of what string values passed to `order` looked like, before adding the complication of when to use Arel, but I'm fine with moving it up to the first description.